Plugins/Detray/include/Acts/Plugins/Detray/DetrayConverter.hpp

```diff
@@ -91,6 +93,19 @@ class DetrayConverter {
       }
     }
 
+    // (3) surface grids
+    if (options.convertSurfaceGrids) {
+      detray::io::detector_grids_payload<std::size_t, detray::io::accel_id>
+          surfaceGridsPayload =
+              DetraySurfaceGridsConverter::convertSurfaceGrids(detector);
+
+      detray::io::surface_grid_reader<typename detector_t::surface_type,
```

Comment:
Just as an explanation for some of the magic numbers detray requires here
```suggestion
      // Capacity 0 (dynamic bin size) and dimension 2 (2D grids)
      detray::io::surface_grid_reader<typename detector_t::surface_type,
```

Reply:
Fixed.